Cannot specify link libraries for target "opencv_gapi"

I used cmake to compile opencv-master and opencv-contrib under windows10. After selecting with-cuda and dnn-cuda and other options, this error occurred, which caused the subsequent compilation to fail.
`CMake Error at modules/gapi/CMakeLists.txt:193 (target_link_libraries):
  Cannot specify link libraries for target "opencv_gapi" which is not built
  by this project.
Call Stack (most recent call first):
  modules/world/CMakeLists.txt:13 (include)
  modules/world/CMakeLists.txt:32 (include_one_module)`

--- a/modules/gapi/CMakeLists.txt
+++ b/modules/gapi/CMakeLists.txt
@@ -190,7 +190,7 @@ endif()
 
 if(WIN32)
   # Required for htonl/ntohl on Windows
-  target_link_libraries(${the_module} PRIVATE wsock32 ws2_32)
+  ocv_target_link_libraries(${the_module} PRIVATE wsock32 ws2_32)
 endif()
 
 ocv_add_perf_tests()

